src/WebJobs.Script/EnvironmentSettingNames.cs

```diff
@@ -23,5 +21,6 @@ public static class EnvironmentSettingNames
         public const string AzureWebJobsExtensionsPath = "AzureWebJobs_ExtensionsPath";
         public const string AzureWebsiteAppCountersName = "WEBSITE_COUNTERS_APP";
         public const string AzureWebJobsSecretStorageType = "AzureWebJobsSecretStorageType";
+        public const string AppInsightsInstrumentationKey = "APPINSIGHTS_INSTRUMENTATIONKEY";
```

Comment:
Ah, yeah.... looks like that's what we were using already. So probably need to keep it...